src/Parser/AVAILABLE_CONFIGS.js

```diff
@@ -21,6 +21,9 @@ import SubtletyRogue from './SubtletyRogue/CONFIG';
 
 import VengeanceDemonHunter from './VengeanceDemonHunter/CONFIG';
 import AfflictionWarlock from './AfflictionWarlock/CONFIG';
+
+import BloodDeathKnight from './BloodDeathKnight/CONFIG';
```

Comment:
The new lines are not necessary here.

Reply:
I am not clear in what you mean? It seems the different specs here are separated into their own little groups. So you just want me to move my DK line up against the afflic lock line?